lib/src/validation-server/server.spec.ts

```diff
@@ -0,0 +1,26 @@
+import request from "supertest";
+import { parse } from "../neu/parser";
+import { runValidationServer } from "./server";
+
+const CONTRACT_PATH = "./lib/src/__examples__/contract.ts";
+
+describe("Server", () => {
+  const mockLogger = {
+    log: (message: string) => message
+  };
+
+  const contract = parse(CONTRACT_PATH);
+
+  describe("Run", () => {
+    it("/health and return 200", done => {
+      const { app } = runValidationServer(5907, contract, mockLogger);
+
+      request(app)
+        .get("/health")
+        .expect(200)
+        .then(_ => {
```

Comment:
This should work too FYI:

```suggestion
        .then(done);
```

Or you could make the test function async, e.g:
```typescript
it(..., async () => {
  const { app } = ...
  await request(app).get("/health").expect(200);
});
```

Reply:
Much better, thanks!

Reply:
The second approach, using `await` doesn't really work, the method is not `async`